ansible/main.yml

```diff
@@ -97,4 +97,6 @@
     - post_software
     - post_software_tasks
 
+- import_playbook: save_output_dir.yml
+
```

Comment:
That playbook is missing in your PR ?

Reply:
Ah! Good catch! I did miss it. Adding it now.